tools/docker/fabric-all-in-one/run-fabric-network.sh

```diff
@@ -21,9 +21,6 @@ function main()
   tar -cC '/etc/hyperledger/fabric/fabric-ccenv/' . | docker load
   tar -cC '/etc/hyperledger/fabric/fabric-tools/' . | docker load
   tar -cC '/etc/hyperledger/fabric/fabric-ca/' . | docker load
-  tar -cC '/etc/hyperledger/fabric/fabric-couchdb/' . | docker load
```

Comment:
@outSH Why move the couchdb image loading to the 2.x branch of the if condition? Does 1.4 not support CouchDB? (I don't know, I haven't looked it up)

Reply:
@petermetz No, Dockerfile for 1.4 doesn't contain couchdb image and container was failing for this reason :)

Reply:
@outSH Ouch, got it. Thank you for the fix!